lib/modules/apostrophe-images/public/css/grid-item.less

```diff
@@ -83,10 +83,6 @@
 
     .apos-transition();
   }
-  input:checked + .apos-field-input-checkbox-indicator,
```

Comment:
Trailing comma typo, but also the rule doesn't do anything either since there is no unit.